calico-vpp-agent/cni/cni_server.go

```diff
@@ -55,6 +58,8 @@ type Server struct {
 	tuntapDriver   *pod_interface.TunTapPodInterfaceDriver
 	vclDriver      *pod_interface.VclPodInterfaceDriver
 	loopbackDriver *pod_interface.LoopbackPodInterfaceDriver
+
+	availableBuffers float64
```

Comment:
Is there a reason not to use uint64 ? Looks like it could reduce the number of casts